source/jormungandr/jormungandr/timezone.py

```diff
@@ -81,3 +81,10 @@ def get_timezone():
     if not hasattr(g, 'timezone'):
         raise TechnicalError("No timezone set for this API")  # the set_request_timezone has to be called at init
     return g.timezone
+
+
+def get_default_timezone():
+    try:
+        return get_timezone()
+    except:
```

Comment:
Seen in sonar: you may prefer (to avoid catch of BaseException and for a "cleaner" code)
```suggestion
    except Exception as _:
```

Reply:
:+1: